cli/commands.go

```diff
@@ -25,7 +25,7 @@ var (
 		{
 			Name:      "download",
 			ShortName: "d",
-			Usage:     "",
+			Usage:     "Downloads the step wit provided --id, and --version, from specified --collection into local step cache. If no --version defined, the latest version of step will be cached.",
```

Comment:
Download the step with provided --id and --version, from specified --collection, into local step downloads cache. If no --version defined, the latest version of the step (latest found in the collection) will be downloaded into the cache.